test/checks/navigation/internal-link-present.js

```diff
@@ -9,7 +9,7 @@ describe('internal-link-present', function () {
 
 	afterEach(function () {
 		fixture.innerHTML = '';
-		axe._tree = undefined;
+		axe._tree = undefined; 
```

Comment:
Sidenote: should we move this `axe._tree` thing to the `checkContext.reset()` method?

Reply:
I think we're better off not storing the flat tree as a global.